features/polyExplorer/src/components/entityList/entityList.jsx

```diff
@@ -98,13 +104,25 @@ function EntityList({ entities, sideLabel }) {
     );
 }
 
+function normalizeEntities(entities) {
+    const validEntities = typeof entities == "object" ? entities : [];
+    return Array.isArray(validEntities)
+        ? { null: validEntities }
+        : validEntities;
+}
+
 export default (props) => {
+    const entities = normalizeEntities(props.entities);
+    const showGrouped =
+        "showGrouped" in props ? props.showGrouped : Object.keys(entities) > 1;
```

Comment:
This shows a bit better what I had in mind with this _preprocessing_:

It can be used like this: `<EntityList entities={[e1, e2, e3]} />`
But also like this: `<EntityList entities={"A": [e1], "B": [e2, e3]}/>`
Or like this: `<EntityList entities={"Whatever: [e3], "Rest": [e1, e2]}/>`

And sensible defaults for `showGroups` and `sideLabel` are chosen in each scenario, while keeping the logic of dealing with this kind of fuzzy input isolated in one place and the component pretty clean.